certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -3,676 +3,2194 @@
+            "type": "string",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "User Password",
+            "description": "Protect the opening of the PDF with the user password"
+          },
+          {
+            "in": "body",
+            "name": "fileData",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FileData"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/FileResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "important",
+        "summary": "Protect PDF",
+        "description": "Protect a PDF with a password"
+      }
+    },
+    "/UnProtectPdf": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "UnProtectPdf",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "header",
+            "name": "ownerPassword",
+            "type": "string",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Owner Password",
+            "description": "Owner Password"
+          },
+          {
+            "in": "body",
+            "name": "fileData",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FileData"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/FileResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "important",
+        "summary": "Unprotect PDF",
+        "description": "Remove the protection from a pdf"
+      }
+    },
+    "/ConvertXmlToJson": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "ConvertXmlToJson",
+        "consumes": [
+          "text/plain",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "XML",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "XML",
+            "description": "XML"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "string"
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
```

Comment:
@stsiek The operation has been hidden. This change breaks the designer for existing flows.

Reply:
we would like to set this action to deprecated, so it is set to "internal". we had understood that it can still be used for existing flows but is no longer visible when adding new actions. We also tested it and it works fine.